doc/Command-Reference.md

```diff
@@ -254,6 +256,10 @@ By default, login takes the user to the default prompt from which all the show c
 
 Go Back To [Beginning of the document](#) or [Beginning of this section](#basic-tasks)
 
+### Show Management Interface
+
+Please check [show ip interfaces](#show-ip-interfaces)
```

Comment:
Does this mean there is no specific command only to show management and loopback interfaces? When these interfaces need to be shown, just call `show ip interfaces` and find the ones of interest?

Reply:
Yes. Do you have any concern?

Reply:
Nope, just to clarify